javatime/src/main/java/drewhamilton/androiddatetimeformatters/javatime/AndroidDateTimeFormatter.java

```diff
@@ -13,22 +14,53 @@
 
 import androidx.annotation.RequiresApi;
 
+import static android.text.format.DateFormat.*;
+
 /**
- * Provides Android-specific {@link DateTimeFormatter}s, such as a localized time formatter that
- * respects the user's 12-/24-hour clock preference.
+ * Provides Android-specific {@link DateTimeFormatter}s, such as a localized time formatter that respects the user's
+ * 12-/24-hour clock preference.
  */
+@RequiresApi(26)
 public final class AndroidDateTimeFormatter {
 
     /**
-     * Returns a {@link DateTimeFormatter} that can format the time according to the context's
-     * locale and the user's 12-/24-hour clock preference.
+     * Returns a {@link DateTimeFormatter} that can format the time according to the context's locale and the user's
+     * 12-/24-hour clock preference.
      * @param context the application context
      * @return a {@link DateTimeFormatter} that properly formats the time.
      */
-    @RequiresApi(26)
     public static DateTimeFormatter ofLocalizedTime(Context context) {
-        DateFormat legacyFormat = android.text.format.DateFormat.getTimeFormat(context);
+        return convertLegacyFormat(context, getTimeFormat(context));
+    }
+
+    /**
+     * Returns a {@link DateTimeFormatter} that can format the date according to the context's locale.
+     * @param context the application context
+     * @return a {@link DateTimeFormatter} that properly formats the date.
+     */
+    public static DateTimeFormatter ofLocalizedDateShort(Context context) {
```

Comment:
Thoughts on exposing just one `ofLocalizedDate` method that accepts a `FormatStyle` as the second parameter? Makes the method name less clunky.

Could map `FormatStyle.FULL` to `LONG` since `FULL` doesn't exist for the legacy format.